readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
-Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+Replace `<rodionsav>` with your Github username and copy the links to Pull Request description:
```

Comment:
I accidentally replaced this place

Reply:
hey! you replaced but not added this links to the PR description. Check 20 point of this [instruction](https://mate-academy.github.io/layout_task-guideline/#how-to-solve-the-layout-tasks-on-github)